infra/repo_manager.py

```diff
@@ -146,6 +146,35 @@ def get_commit_list(self, old_commit, new_commit):
     commits.append(old_commit)
     return commits
 
+  def _get_git_history(self):
```

Comment:
I find this method confusing because "getter" methods usually return something, not modify state.
Can you call this `fetch_unshallow`?

Reply:
Fixed.